test/src/test-uv-link-source-t.c

```diff
@@ -113,7 +110,9 @@ static void close_cb(uv_link_t* link) {
 TEST_IMPL(uv_link_source_t) {
   CHECK_NE(loop = uv_default_loop(), NULL, "uv_default_loop()");
 
+  #ifndef _WIN32
   CHECK_EQ(socketpair(AF_UNIX, SOCK_STREAM, 0, fds), 0, "socketpair()");
```

Comment:
Well, it will compile, but the test won't work at all :) Let's `ifndef` whole test implementation